packages/network/src/createSyncWorker.ts

```diff
@@ -14,13 +12,13 @@ import { Output } from "./workers/SyncWorker";
  * dispose: function to dispose of the sync worker
  * }
  */
-export function createSyncWorker<Cm extends Components>() {
-  const config$ = new Subject<SyncWorkerConfig<Cm>>();
+export function createSyncWorker() {
+  const config$ = new Subject<SyncWorkerConfig>();
   const worker = new Worker(new URL("./workers/Sync.worker.ts", import.meta.url), { type: "module" });
-  const ecsEvent$ = new Subject<NetworkComponentUpdate<Cm>>();
+  const ecsEvent$ = new Subject<NetworkEvent>();
 
   // Pass in a "config stream", receive a stream of ECS events
-  const subscription = fromWorker<SyncWorkerConfig<Cm>, Output<Cm>>(worker, config$).subscribe(ecsEvent$);
+  const subscription = fromWorker<SyncWorkerConfig, NetworkEvent>(worker, config$).subscribe(ecsEvent$);
```

Comment:
I see `NetworkEvent` accepts a `Components` generic type - could we keep the generic typing for `createSyncWorker` and pass it to `NetworkEvent`? Makes working with `ecsEvent$` slightly nicer